docs/whatsnew/5.2.rst

```diff
@@ -51,6 +51,22 @@ cosmologies with their non-flat equivalents.
     True
 
 
+Cosmology instance can be parsed from or converted to a HTML table using
+the new HTML methods in Cosmology's ``to/from_format`` I/O.
+
+    >>> from astropy.cosmology import Planck18
+    >>> Planck18.write("planck18.html")
+
+If the following is added to the file's header, the column names will render
+nicely.
+::
+    <script
```

Comment:
This doc makes me wonder: should it be an option to have the column names as latex? After all, the html looks OK without it if you keep the regular column names.

Reply:
I agree. That only needs to be a flag in ``write_html_table``.

Reply:
And for a followup PR, it would be good to move the flag into ``to/from_table``.